immunopepper/spark.py

```diff
@@ -138,26 +96,16 @@ def filter_whitelist(matrix, name_list, index_name, jct_annot_col, rf_annot_col)
             matrix = matrix.filter(sf.abs(sf.hash('kmer') % tot_batches) == batch_id)
 
         # Filter on junction status depending on the content on the matrix
-        if cross_junction:
-            matrix = matrix.filter(f'{jct_col} == True')
-        else:
-            matrix = matrix.filter(f'{jct_col} == False')
         matrix = matrix.drop(jct_col)
 
-        # Cast type to double
-        matrix = cast_type_dbl(matrix, index_name, jct_annot_col, rf_annot_col)
-
-        # Fill Nans
-        logging.info("Remove Nans")
-        matrix = matrix.na.fill(0)
 
         # Separate kmers only present in the backbone annotation from the ones supported by the reads of any sample
-        partitions_ = matrix.rdd.getNumPartitions()
-        logging.info(f'...partitions: {partitions_}')
-        if separate_back_annot:
-            logging.info("Isolating kmers only in backbone annotation")
-            matrix = split_only_found_annotation_backbone(separate_back_annot, output_dir, matrix, index_name,
-                                                          jct_annot_col, rf_annot_col, cross_junction)
+#        partitions_ = matrix.rdd.getNumPartitions()
```

Comment:
why only comment and not remove? It is still present in the history, if you need it back